tmk_core/protocol/lufa.mk

```diff
@@ -67,10 +67,8 @@ LUFA_OPTS += -DFIXED_CONTROL_ENDPOINT_SIZE=8
 LUFA_OPTS += -DFIXED_CONTROL_ENDPOINT_SIZE=8
 LUFA_OPTS += -DFIXED_NUM_CONFIGURATIONS=1
 
-# Remote wakeup fix for ATmega32U2        https://github.com/tmk/tmk_keyboard/issues/361
-ifeq ($(MCU),atmega32u2)
-	LUFA_OPTS += -DNO_LIMITED_CONTROLLER_CONNECT
-endif
+# Remote wakeup fix for ATmega32U2 (and also some other series 2 and xmega chips)        https://github.com/tmk/tmk_keyboard/issues/361
+LUFA_OPTS += -DNO_LIMITED_CONTROLLER_CONNECT
```

Comment:
```suggestion
# Remote wakeup fix for ATmega16/32U2        https://github.com/tmk/tmk_keyboard/issues/361
ifneq (,$(filter $(MCU), at90usb162 atmega16u2 atmega32u2))
	LUFA_OPTS += -DNO_LIMITED_CONTROLLER_CONNECT
endif
```
It's a good idea to keep the number of defines to a minimum anyway.

Reply:
Alright, I'm fine with that, the only MCUs we are skipping this way are: 8K MCUs, which are never gonna be supported because of their size, and XMEGA MCUs, which is a class of devices we don't seem to currently support.